src/os/bluestore/BlueStore.cc

```diff
@@ -8321,8 +8321,15 @@ void BlueStore::collect_metadata(map<string,string> *pm)
   bdev->collect_metadata("bluestore_bdev_", pm);
   if (bluefs) {
     (*pm)["bluefs"] = "1";
+    // this computed value is for backward compatibility only
+    bool single_shared_device = \
+      !bluefs_layout.dedicated_db && !bluefs_layout.dedicated_wal;
```

Comment:
nit: make single_shared_device() helper in bluefs_layout_t?

Reply:
Makes sense.

Reply:
Added.